.github/workflows/config/nodejs-e2e.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I initially wanted yaml, but there's no yaml in Go's standard library that I know of. We would have to use [`gopkg.in/yaml.v3`](https://pkg.go.dev/gopkg.in/yaml.v3). [`toml`](https://github.com/BurntSushi/toml) would also be a third party library.

The only other option not requiring third party libraries would be rolling our own simple config file format, but now we would have to maintain a config file parser, even if it's a simple `ini`-like format.

If you have suggestions, please let me know.

Reply:
Another idea could be having a `README.md` file in that `config/` directory where we explain the schema for the files.

Reply:
I'm not sure if there's a json5 implementation, but there might be some third party library.

However, after some discussion with Tony, we only need to support simple comments. Comments can easily be stripped as a preprocessing step with some simple regular expressions. So I'll do that, it seems like the simplest option without adding third party depdendencies.

Reply:
I'll only add support for `// single line comments` since those are easy enough. `/* multi line comments */` have some edge cases and the regex needs a 1 character lookahead plus multi-line regexes can get hard to maintain.

Reply:
Last push supports JSON with comments